### ISSUE TITLE
Cannot import name 'get_version' from 'rosetta'

### ISSUE THREAD

**Issue by calenwu** [mbi/django-rosetta#209]
- Which version of Django are you using?: 2.1.2
- Which version of django-rosetta are you using?: 0.9.0

When I run django I get this error message

```
  File "C:\Users\georg\AppData\Local\Programs\Python\Python37\lib\site-packages\rosetta\urls.py", line 8, in <module>
    from . import views
  File "C:\Users\georg\AppData\Local\Programs\Python\Python37\lib\site-packages\rosetta\views.py", line 32, in <module>
    from rosetta import get_version as get_rosetta_version
ImportError: cannot import name 'get_version' from 'rosetta'
```

**Comment by laibulle** [comment]
I have the same issue, did you find any solution ?

**Comment by mbi** [comment]
@laibulle are you also experiencing this issue on a Windows machine?

**Comment by calenwu** [comment]
@mbi you didn't ask me, but I have this issue on both windows and linux (ubuntu 18.10)

**Comment by mbi** [comment]
Hmm… which version of Python, please?

**Comment by calenwu** [comment]
On windows it is 3.7 and ubuntu it is 3.6.x

**Comment by mbi** [comment]
Hmm, I'm testing with exactly the same environment, but can't reproduce this. How did you install Rosetta? Did you e.g. `pip install django-rosetta`, or did you download a zip and ran `python setup.py install`?

**Comment by calenwu** [comment]
I installed it with pip install django-rosetta

**Comment by mbi** [comment]
Could you please test the latest HEAD? 

```
pip uninstall django-rosetta
pip install git+https://github.com/mbi/django-rosetta.git
```

**Comment by calenwu** [comment]
Oh wow it works now, thank you so much!

**Comment by mbi** [comment]
Ah fantastic, thanks for testing! I'll release 0.9.2 shortly

**Comment by mbi** [comment]
Cheers, done. This should be fixed in 0.9.2

**Comment by capiscuas** [comment]
> ```
> pip install git+https://github.com/mbi/django-rosetta.git
> ```

Thanks, this also worked for me, I am having ubuntu with python 3.6 and I got the same error, with other machine with python 3.5 it worked fine with just:
pip install django-rosetta
